spec/views/competitions/my_competitions.html.erb_spec.rb

```diff
@@ -23,7 +23,7 @@
 
   it "shows you are on the waiting list" do
     render
-    expect(rendered).to match 'You are currently on the waiting list'
+    expect(rendered).to match 'Your registration has not been looked at by the organizers or is incomplete. Please check the competition page for details This competition is not confirmed and not visible.'
```

Comment:
There seems to be a copy/paste whoopsie going on here? Superfluous `This competition is not confirmed and not visible.` dangling at the end...